### PR TITLE
create ability to switch languages

### PR DESCRIPTION
**Pull request by yehorDiakonov**

### CODE REVIEW

File: src/localize/index.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  public get(key: string) {
+    return i18next.t(key);
+  }
+
+  public changeLanguage = (language: Languages) => {
+    i18next.changeLanguage(language);
+  };
+}
+
+export const leng = new Locale(Languages.en, {
```
Comment (BogdanKusch) [inline src/localize/index.tsx:30]: lets define interface for this class and export this leng instance as ```export const leng: Localisation = new Locale(...)```

File: src/localize/ru.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+export const ru = {
```
Comment (BogdanKusch) [inline src/localize/ru.ts:1]: lets define json file for this.

File: src/localize/index.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  public get(key: string) {
+    return i18next.t(key);
+  }
+
+  public changeLanguage = (language: Languages) => {
+    i18next.changeLanguage(language);
+  };
+}
+
+export const leng: Language = new Locale(Languages.en, {
```
Comment (BogdanKusch) [inline src/localize/index.tsx:32]: leng - wrong name
localisation, locale, localeService ...

File: src/localize/index.tsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+interface Language {
+  get(key: string): string;
+  changeLanguage(language: string): void;
+}
+
+class Locale {
+  constructor(lng: Languages, resources: {[key in Languages]: any}) {
+    i18next.use(initReactI18next).init({
+      lng,
+      debug: true,
```
Comment (BogdanKusch) [inline src/localize/index.tsx:18]: debug even for production build?

File: src/localize/index.tsx [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+  }
+
+  public changeLanguage = (language: Languages) => {
+    i18next.changeLanguage(language);
+  };
+}
+
+export const leng: Language = new Locale(Languages.en, {
+  [Languages.en]: {
+    translation: require('./en.json'),
```
Comment (BogdanKusch) [inline src/localize/index.tsx:34]: you can import it by using import from latest ES.

File: src/localize/ru.json [hunk 6]
```diff
@@ -0,0 +1,19 @@
+{
+  "title": "Авторизация",
+  "loginPlaceHolder": "введите логин",
+  "passwordPlaceHolder": "введите пароль",
+  "invalidLoginMess": "не верный логин",
+  "invalidPasswordMess": "не правильный пароль",
```
Comment (BogdanKusch) [inline src/localize/ru.json:6]: You can create object here
"errors":  {
 "worngPassword" : "..."
 ....
 }

Comment (BogdanKusch) [reply src/localize/ru.json:6]: We can split localisation data in logical block, for example: errors, etc...

File: src/localize/ru.json [hunk 7]
```diff
@@ -0,0 +1,19 @@
+{
+  "title": "Авторизация",
```
Comment (BogdanKusch) [inline src/localize/ru.json:2]: not clear name when we use it...

**Review by BogdanKusch — CHANGES_REQUESTED** [review]

File: src/localize/index.tsx (hunk 4 above)
Comment (yehorDiakonov) [reply src/localize/index.tsx:18]: i'll delete this

Comment (yehorDiakonov) [reply src/localize/index.tsx:18]: because works without this string

File: src/localize/index.tsx [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+      resources,
+    });
+  }
+
+  public get(key: string, options?: {}) {
+    if(options) {
+      return i18next.t(key, options);
+    } else {
+      return i18next.t(key);
+    }
```
Comment (BogdanKusch) [inline src/localize/index.tsx:29]: i18next.t(key, options); no need to use if statement.

File: src/localize/index.tsx [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+
+class Locale {
+  constructor(lng: Languages, resources: {[key in Languages]: any}) {
+    i18next.use(initReactI18next).init({
+      lng,
+      resources,
+    });
+  }
+
+  public get(key: string, options?: {}) {
```
Comment (BogdanKusch) [inline src/localize/index.tsx:24]: {} type is not right because this object can have properties and string values.

**Review by BogdanKusch — CHANGES_REQUESTED** [review]

File: src/localize/index.tsx [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+import en from './en.json';
+import ru from './ru.json';
+
+export enum Languages {
+  en = 'en',
+  ru = 'ru',
+}
+
+interface Language {
+  get(key: string, options?: {}): string;
```
Comment (BogdanKusch) [inline src/localize/index.tsx:12]: options arg type?

**Review by BogdanKusch — CHANGES_REQUESTED** [review]

**Review by BogdanKusch — APPROVED** [review]

**Pull request merged**